README.md

```diff
@@ -1,31 +1,19 @@
 # epr-re-ex-engineering-prototype
 
-`Node.js` prototype template, using the [GOV.UK Prototype Kit](https://github.com/alphagov/govuk-prototype-kit) and
-the [GOV.UK Frontend](https://github.com/alphagov/govuk-frontend).
-
-> Basically the `GOV.UK Prototype Kit` and `GOV.UK Frontend` wrapped up and provided on the Core Delivery Platform
+A lightweight application allowing engineers working on the `epr-re-ex` services to perform exploratory/protyping activities.
 
+<!-- TOC -->
 - [Requirements](#requirements)
   - [Node.js](#nodejs)
+- [Start the app](#start-the-app)
 - [GOV.UK Prototype Kit and GOV.UK Frontend](#govuk-prototype-kit-and-govuk-frontend)
-- [Using the refreshed GOV.UK brand](#using-the-refreshed-govuk-brand)
-- [Setting a password](#setting-a-password)
-- [Setting multiple passwords](#setting-multiple-passwords)
-- [Removing the need for a password](#removing-the-need-for-a-password)
-- [Npm scripts](#npm-scripts)
-- [Updating dependencies](#updating-dependencies)
-- [Environment Variables and Secrets](#environment-variables-and-secrets)
-  - [Local development](#local-development)
-  - [Environment Variables on CDP](#environment-variables-on-cdp)
-  - [Environment Variables in the GOV.UK Prototype Kit](#environment-variables-in-the-govuk-prototype-kit)
-  - [Secrets](#secrets)
-- [Creating a secret](#creating-a-secret)
 - [Docker](#docker)
   - [Development image](#development-image)
   - [Production image](#production-image)
   - [Debug docker](#debug-docker)
 - [Licence](#licence)
   - [About the licence](#about-the-licence)
+<!-- TOC -->
```

Comment:
TOC's get mangled by prettier, I've found [this works well](https://github.com/DEFRA/epr-re-ex-admin-frontend/pull/12#discussion_r2371780986)